### PR TITLE
fix(ingestion): Only apply python restriction on looker module

### PR DESCRIPTION
**Pull request by treff7es**
## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Comment by anshbansal** [comment]
@treff7es We would need changes in the github workflows also. If you search for `3.9.9` in the repository you will find restrictions added in multiple github workflows files in `.github` folder.

**Comment by treff7es** [comment]
> @treff7es We would need changes in the github workflows also. If you search for `3.9.9` in the repository you will find restrictions added in multiple github workflows files in `.github` folder.

We have Looker test which only runs on python <= 3.9.9 as it is still a restriction but limits it to a module and not everything.

File: metadata-ingestion/src/datahub/ingestion/source/looker.py [hunk 1]
```diff
@@ -1,3 +1,9 @@
+import sys
+
+# restrict python to <=3.9.9 due to https://github.com/looker-open-source/sdk-codegen/issues/944
+if (sys.version_info.major, sys.version_info.minor, sys.version_info.micro) > (3, 9, 9):
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/looker.py:4]: Can make this slightly simpler:

```suggestion
if sys.version_info > (3, 9, 9):
```

**Review by hsheth2 — APPROVED** [review]
Looks good

Quick question: are there any 3.10.x versions that also work?

**Comment by treff7es** [comment]
This pr is obsoloted by this pr:
https://github.com/linkedin/datahub/pull/4312

**Pull request closed without merging**